src/routes/_components/compose/ComposeBox.html

```diff
@@ -43,10 +43,6 @@ <h1 class="sr-only">Compose status</h1>
     max-width: calc(100vw - 40px);
   }
 
-  .compose-box.direct-reply {
-    background-color: var(--status-direct-background);
-  }
-
```

Comment:
I think if we remove this, then it's not very obvious which messages are DMs and which aren't, when we're not in the "conversations" timeline.

I think maybe this color could be set for DMs only when we're _not_ in the conversations timeline.